blockstore/apps/bundles/links.py

```diff
@@ -11,12 +11,18 @@
 import attr
 
 
+def convert_to_uuid(value):
+    if not isinstance(value, UUID):
+        return UUID(value)
+    return value
```

Comment:
Also, this is a not, but how about avoiding the negation here?

```suggestion
    if isinstance(value, UUID):
        return value
    return UUID(value)
```

Reply:
Good suggestion @gabor-boros , done in [054ab81](https://github.com/openedx/blockstore/pull/149/commits/054ab812bfcbcb3613b4786d1a70b37fd3d56414).